code/game/machinery/suit_storage_unit.dm

```diff
@@ -305,6 +305,9 @@
 
 /obj/machinery/suit_storage_unit/attackby(obj/item/I, mob/user, params)
 	if(state_open && is_operational())
+		if(istype(I, /obj/item/clothing/head/mob_holder))
+			to_chat(user, "<span class='warning'>That doesn't fit in the unit like that! Try another method!</span>")

```

Comment:
This is gross, and snowflakey. This should be expanded to be something that is either a flag or a var on an object that, if set, prevents it from being placed inside this.